cmd/base64_decoder/decoder.go

```diff
@@ -0,0 +1,62 @@
+package main
+
+import (
+	"encoding/json"
+	"flag"
+	"fmt"
+
+	"github.com/hyperledger-labs/orion-server/pkg/types"
+)
+
+var help = "The decoder decodes the base64 encoded value field in the " +
+	"GetDataResponseEnvelope. Pass the json output of " +
+	"GetDataResponseEnvelope to the `-getresponse` flag"
+
+type GetDataResponseEnvelope struct {
+	Response  *GetDataResponse `json:"response,omitempty"`
+	Signature []byte           `json:"signature,omitempty"`
+}
```

Comment:
If you refrain from printing the signature, you can get away from redefining this, no?

Reply:
I know it is imperfect but removing the envelope field requires more unnecessary descriptions to this utility and documentation. On the other hand, this signature is invalid on the decoded data. 

I would like to keep this simple. The decoder just takes the `Envelope` and decodes the `value` field, output the envelope again with the decoded value field.

Reply:
ok